python/dgl/sparse/matmul.py

```diff
@@ -229,56 +229,113 @@ def spspmm(
     )
 
 
-def mm(
-    A1: Union[SparseMatrix, DiagMatrix],
+def matmul(
+    A1: Union[torch.Tensor, SparseMatrix, DiagMatrix],
```

Comment:
Use A B, instead of A1, A2 to keep consistency with https://github.com/dmlc/dgl/blob/master/python/dgl/sparse/elementwise_op.py?

Reply:
done